packages/template-retail-react-app/app/translations/README.md

```diff
@@ -1,60 +1,25 @@
-# Translations & Localization
+# Translations
 
 Most of the files in this folder are generated by `react-intl` **CLI tool**:
 
--   `pwa/app/translations/en-GB.json` <- output of _extracting_ the default messages, which you can send to your translators
--   `pwa/app/translations/[locale].json` <- the files that your translators make for the other locales
--   `pwa/app/translations/compiled/[locale].json` <- output of _compiling_ the messages into AST format
-    -   Compiling helps improve the performance because it allows `react-intl` to skip the parsing step
+- `app/translations/en-US.json` <- output of _extracting_ the default messages, which you can send to your translators. 
+  - By default, we chose en-US because it is the locale of the app's `defaultMessage`s
+- `app/translations/[locale].json` <- the files that your translators make for the other locales
+- `app/translations/compiled/[locale].json` <- output of _compiling_ the messages into AST format
+  - Compiling helps improve the performance because it allows `react-intl` to skip the parsing step
```

Comment:
```suggestion
- Compiling helps improve the performance because it allows `react-intl` to skip the parsing step
```

Reply:
I added the indentation so that the line item is a sub-bullet. So yeah, I meant it to be indented like that.